Source/processcontainers/common/NetworkInfoUnimplemented.h

```diff
@@ -79,24 +79,15 @@ namespace ProcessContainers {
             return 0;
```

Comment:
Let's throw away this file completely and just return nullptr when NetworkInfo is not available for a Container (more like we do with com). (and adjust the plugin if needed so it deals with this case)

Reply:
Done.